types/vimeo__player/index.d.ts

```diff
@@ -1,10 +1,11 @@
-// Type definitions for @vimeo/player 2.9.1
+// Type definitions for @vimeo/player 2.10.0
```

Comment:
```suggestion
// Type definitions for @vimeo/player 2.10
```
https://github.com/definitelyTyped/DefinitelyTyped/#how-do-definitely-typed-package-versions-relate-to-versions-of-the-corresponding-library